src/ILCompiler/repro/Program.cs

```diff
@@ -3,11 +3,28 @@
 // See the LICENSE file in the project root for more information.
```

Comment:
All the temp changes in repro/Program.cs would be nice to rebase away.

Reply:
Done